package.json

```diff
@@ -37,7 +37,8 @@
     "lint": "tsc --noEmit && eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0 && npm run lint:styles && npm run lint:md",
     "lint:styles": "stylelint \"src/**/*.scss\" --fix",
     "lint:md": "markdownlint \"src/**/*.{md,mdx}\" --fix",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "icons:format": "npx @svgr/cli --out-dir src/assets/formatted --ignore-existing -- src/assets"
```

Comment:
we should use the typescript option so it generates the interfaces too [TypeScript](https://react-svgr.com/docs/options/#typescript)

Reply:
https://react-svgr.com/docs/options/#accessibility

Reply:
I don't see any harm in using these two options too: https://react-svgr.com/docs/options/#ref  https://react-svgr.com/docs/options/#memo might future proof us from other usecases or performance issues

Reply:
I initially was using the tsx option, but the app was having trouble parsing the components for some reason and I couldn't figure out exactly what the issue was. I think there's probably some issue with the `babel` config that svgr uses, so I'll play around with that and see if I can figure it out. 

I can definitely add title, role and memoize the component

Reply:
Ok, I was able to resolve the `tsx` issue by writing a custom template in the svgr config and fixing the imports/typing etc. I also added `role: 'img'` to the svgProps and `forwardRef` and `memo` to the template